macros/cross_db_utils/_is_relation.sql

```diff
@@ -0,0 +1,5 @@
+{% macro _is_relation(obj, macro) %}
```

Comment:
This is the intended behavior, but it's important to note: This will raise if the `ref` of an ephemeral model is passed in here. The `ref()` function returns a string for ephemeral models, vs. a Relation for any non-ephemeral model (or source). I believe all of the macros here that use `_is_relation` are expecting a table/view, so that is A-OK!